spec/rubocop/cop/rspec/rails/have_http_status_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Rails::HaveHttpStatus do
+  it 'registers an offense for `expect(response.status).to be 200`' do
+    expect_offense(<<~RUBY)
+      it { expect(response.status).to be 200 }
+           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `expect(response.status).to be 200` over `expect(response).to have_http_status(200)`.
```

Comment:
I think this message suggests the other way around.

Reply:
Oops, I'll fix it.

Reply:
Fixed in https://github.com/rubocop/rubocop-rspec/pull/1307/commits/2cd3a316919b8d973125ec8ea8aabe53858d45b2